src/routes/router.js

```diff
@@ -0,0 +1,11 @@
+import express from "express";
+
+const router = express.Router();
+
+
+
+router.get('/', (req, res) =>{
+    res.send("Welcome!")
+});
+
+export default router;
```

Comment:
In the routes folder, let's rename the file by `*.route.js`. For example, If you are implementing authentication routes, you can name it `authentication.route.js`